### PR TITLE
add support for importing protected branches and tags

### PR DESCRIPTION
**Pull request by aquavitae**

### CODE REVIEW

**Comment by armsnyder** [comment]
Can you modify the tests to add a TestStep with `ImportState: true`?

**Comment by armsnyder** [comment]
@aquavitae Are you still working on this PR?

File: gitlab/resource_gitlab_branch_protection.go [hunk 1]
```diff
@@ -17,6 +17,9 @@ func resourceGitlabBranchProtection() *schema.Resource {
 		Create: resourceGitlabBranchProtectionCreate,
 		Read:   resourceGitlabBranchProtectionRead,
 		Delete: resourceGitlabBranchProtectionDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```
Comment (timofurrer) [inline gitlab/resource_gitlab_branch_protection.go:21]:
```suggestion
			StateContext: schema.ImportStatePassthroughContext,
```

File: gitlab/resource_gitlab_tag_protection.go [hunk 2]
```diff
@@ -17,6 +17,9 @@ func resourceGitlabTagProtection() *schema.Resource {
 		Create: resourceGitlabTagProtectionCreate,
 		Read:   resourceGitlabTagProtectionRead,
 		Delete: resourceGitlabTagProtectionDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```
Comment (timofurrer) [inline gitlab/resource_gitlab_tag_protection.go:21]:
```suggestion
			StateContext: schema.ImportStatePassthroughContext,
```

**Review by timofurrer — CHANGES_REQUESTED** [review]
Please rebase to latest master for SDK v2 changes and pls accept the suggestions. Otherwise, LGTM.

**Comment by armsnyder** [comment]
Both of these importers were also added in #318 so I'm going to close this PR. Thanks!

**Pull request closed without merging**